app/controllers/worldwide_organisations_controller.rb

```diff
@@ -23,8 +23,36 @@ def show
     end
   end
 
+  def show_b_variant
+    @world_location = WorldLocation.with_translations(I18n.locale).find(params[:world_location_id])
+    @embassy_data = embassy_test_data(params[:id])
+    @primary_role = primary_role
+    @other_roles = ([secondary_role] + office_roles).compact
+
+    set_meta_description(@worldwide_organisation.summary)
+    set_slimmer_organisations_header([@worldwide_organisation] + @worldwide_organisation.sponsoring_organisations)
+
+    if show_b_variant? && @embassy_data
```

Comment:
I think if we use `is_under_test?` we could have a situation where we have, for example, an `organisation` in a `world_location` where the `world_location` is in the YAML file, but the `organisation` isn't.....in that instance `is_under_test?` would return true (because it's testing for existence of the `world_location` in the YAML) but there would be no content.

For example - if our YAML was

```
australia:
  taxonomy:
    - title: Help for British nationals in Australia
      description: Travel documents and help with emergencies.
      base_path: help-for-british-nationals
      tagged_content:
        - title: Get a new or replacement UK passport
          description: Forms, prices and application
          base_path: /apply-renew-passport
        - title: Get emergency UK travel documents
          description: If you're abroad, need to travel and can't get a passport in time.
          base_path: /emergency-travel-document
  embassies:
    british-high-commission-canberra:
      - title: British High Commission Canberra
        summary: The summary
        body: |
          The High Commission in Canberra....
brazil:
  taxonomy:
.
.
.
```

and we hit `/world/australia/british-high-commission-melbourne`, we would end up with `is_under_test?` returning true because Australia is in the YAML, but there would be nothing for the `british-high-commission-melbourne`

I did think of pushing the test for having actual embassy data into the test helper, but I thought it might get messy to start iterating through the YAML structure looking for data so I left it in the controller for now.